src/constants.py

```diff
@@ -0,0 +1,19 @@
+API_URL = '{base_url}/predApi/v1.0/deployments/{deployment_id}/predictions'  # noqa
```

Comment:
This constant doesn't seem to be used in the application. Is there an intention to do so at some point or can we remove it?

Reply:
I was wondering if we should provide a basic predict function, but on second thought we don't really know what users would try to include in their request :thinking: